pydis_site/apps/api/viewsets/bot/user.py

```diff
@@ -284,6 +317,48 @@ def bulk_patch(self, request: Request) -> Response:
 
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+    @action(detail=True, methods=['POST'], name="Add alternate account",
+            url_name='alts', url_path='alts')
+    def add_alt(self, request: Request, pk: str) -> Response:
+        """Associate the given account to the user's alternate accounts."""
+        user = self.get_object()
+        if not isinstance(request.data, int):
+            raise ParseError(detail={"non_field_errors": ["Request body should be an integer"]})
+
+        try:
+            alt = User.objects.get(id=request.data)
+        except ObjectDoesNotExist:
+            raise ParseError(detail={"non_field_errors": ["Unknown alternate account ID"]})
+        try:
+            user.alts.add(alt)
+        except IntegrityError as err:
+            if err.__cause__.diag.constraint_name == 'api_useraltrelationship_prevent_alt_to_self':
+                raise ParseError(detail={
+                    "non_field_errors": ["The user may not be an alternate account of itself"]
+                })
+
+            raise  # pragma: no cover
+
+        serializer = self.get_serializer(instance=user)
+        return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+    # See https://www.django-rest-framework.org/api-guide/viewsets/#routing-additional-http-methods-for-extra-actions
+    @add_alt.mapping.delete
+    def remove_alt(self, request: Request, pk: str) -> Response:
+        """Disassociate the given account from the user's alternate accounts."""
+        user = self.get_object()
+        if not isinstance(request.data, int):
+            raise ParseError(detail={"non_field_errors": ["Request body should be an integer"]})
+        try:
+            alt = user.alts.get(id=request.data)
+        except ObjectDoesNotExist:
+            raise ParseError(detail={
+                'non_field_errors': ["Specified account is not a known alternate account of this user"]
+            })
+
+        user.alts.remove(alt)
+        return Response('', status=status.HTTP_204_NO_CONTENT)
```

Comment:
```suggestion
        return Response(status=status.HTTP_204_NO_CONTENT)
```
The empty string here seemed to make aiohttp sad when consuming this response. Removing this seemed to solve it.

Reply:
Strange. I removed it.